ex1.py

```diff
@@ -0,0 +1,22 @@
+import collections
+
+def compter_lettre(word, letter):
+	i = word.count(letter)
+	print(f'dans le mot "{word}" il y a "{i}" fois la lettre "{letter}"')
+	
+def compter_lettres(word):
+	i  = {
+		x: word.count(x) 
+		for x in word
+		
```

Comment:
You should clean more your code before sending it to review. Normalize your code, remove unnecessary empty lines/spaces/comments. Add necessary empty lines/spaces/comments